### PR TITLE
No double intercepts when using a workflow

### PR DESCRIPTION
**Pull request by hfrick**
tests for https://github.com/tidymodels/parsnip/pull/1033

### CODE REVIEW

File: .github/workflows/GH-R-CMD-check.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
           try(pak::pkg_install("tidymodels/hardhat"))
           try(pak::pkg_install("tidymodels/modeldata"))
           try(pak::pkg_install("tidymodels/multilevelmod"))
-          try(pak::pkg_install("tidymodels/parsnip"))
+          try(pak::pkg_install("tidymodels/parsnip#1033"))
```
Comment (hfrick) [inline .github/workflows/GH-R-CMD-check.yaml:69]: TODO: point back to `main` before merging

**Comment by topepo** [comment]
The parsnip PR is merged.

**Comment by hfrick** [comment]
I'll update the GHA file and merge this one too!

File: .github/workflows/GH-R-CMD-check.yaml [hunk 2]
```diff
@@ -66,7 +66,7 @@ jobs:
           try(pak::pkg_install("tidymodels/hardhat"))
           try(pak::pkg_install("tidymodels/modeldata"))
           try(pak::pkg_install("tidymodels/multilevelmod"))
-          try(pak::pkg_install("tidymodels/parsnip"))
+          try(pak::pkg_install("tidymodels/parsnip#1035"))
```
Comment (hfrick) [inline .github/workflows/GH-R-CMD-check.yaml:69]: TODO: point back to main before merging this PR

File: .github/workflows/spark-R-CMD-check.yaml [hunk 3]
```diff
@@ -68,7 +68,7 @@ jobs:
 
       - name: Install devel versions
         run: |
-          try(pak::pkg_install("tidymodels/parsnip"))
+          try(pak::pkg_install("tidymodels/parsnip#1035"))
```
Comment (hfrick) [inline .github/workflows/spark-R-CMD-check.yaml:71]: TODO: point back to main before merging this PR

**Pull request merged**